docs/bbe-guidelines/bbe-guidelines.md

```diff
@@ -0,0 +1,174 @@
+# Guidelines for Writing Ballerina By Examples (BBEs)
+
+## General Best Practices
+
+1. The maximum character count per line should be 80 in .bal files. Otherwise, the lines will be wrapped in the [Ballerina](https://ballerina.io/) website reducing the readability.
+
+Example:
+
+
+
+
+<p id="gdcalert1" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/Guidelines-in0.png). Store image on your image server and adjust path/filename if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert2">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
+
+
+![alt_text](images/Guidelines-in0.png "image_tooltip")
+
+
+
+
+*   **Add comments to the code blocks as much as possible with “//”. They will move to the RHS box in b.io. So comments are used as a mechanism to describe the code.**
```

Comment:
Who's the intended audience of this doc? If this is applicable to everyone, then we should probably minimize the use of abbreviations such as "b.io"

Reply:
Changed as "[Ballerina website](https://ballerina.io/)".